completions/rsync

```diff
@@ -1,5 +1,35 @@
 # bash completion for rsync                                -*- shell-script -*-
 
+_comp_cmd_rsync__vercomp()
+{
+    if [[ "$1" == "$2" ]]; then
+        return 0
+    fi
+    local IFS=.
+    local i ver1=($1) ver2=($2)
+    # fill empty fields in ver1 with zeros
+    for ((i = ${#ver1[@]}; i < ${#ver2[@]}; i++)); do
+        ver1[i]=0
+    done
```

Comment:
I prefer determining the max number first to filling zeroes.
```suggestion
    local i ver1=($1) ver2=($2)
    local n=$((${#ver1[@]} >= ${#ver2[@]} ? ${#ver1[@]} : ${#ver2[@]}))
```

Or another way might be to loop with `while [[ ${ver1[i]-}${ver2[i]-} ]]; do ... ; ((i++)); done`